bedrock/firefox/templates/firefox/channel/ios.html

```diff
@@ -9,8 +9,6 @@
 {% block page_title %}
 {% if l10n_has_tag('firefox_title_meta_bug1345338') %}
   {{ _('Try New Features in a Pre-Release iOS Browser | Firefox') }}
-{% else %}
-  {{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.')}}
```

Comment:
Agreed, it's not worth spending more time on. This is good. Thanks for your patience!